test/dsets.c

```diff
@@ -15719,11 +15718,7 @@ main(void)
                 nerrors += (test_multiopen(file) < 0 ? 1 : 0);
                 nerrors += (test_types(file) < 0 ? 1 : 0);
                 nerrors += (test_userblock_offset(envval, my_fapl, new_format) < 0 ? 1 : 0);
-
-                if (!driver_uses_modified_filename) {
```

Comment:
These places intentionally check if the driver uses a modified filename so that they aren't run for certain VFDs.